react/src/components/tabs/components/SprkTabsButton/SprkTabsButton.js

```diff
@@ -101,11 +105,16 @@ SprkTabsButton.propTypes = {
    * The id used for the data-id attribute
    */
   tabBtnDataId: PropTypes.string,
+  // TODO remove as part of Issue 3566
   /**
-   *
-   The value used for the data-analytics attribute
+   * Deprecated - use `tabBtnAnalyticsString` instead. The value used for
+   * the data-analytics attribute.
    */
   tabBtnAnalytics: PropTypes.string,
+  /**
+   * The value used for the data-analytics attribute.
```

Comment:
Lets use the same description here that we standardized on in the other components.

Reply:
hey @bobwise  looks like you missed addressing this comment. Here's the description we should use that I copied from another component.
"Assigned to the `data-analytics` attribute serving as a unique selector for outside libraries to capture data."